crates/biome_js_analyze/src/lint/nursery/no_adjacent_overload_signatures.rs

```diff
@@ -0,0 +1,332 @@
+use biome_analyze::{context::RuleContext, declare_rule, Ast, Rule, RuleDiagnostic, RuleSource};
+use biome_console::markup;
+use biome_js_syntax::{
+    AnyJsClassMember, AnyJsModuleItem, AnyJsStatement, JsExport, JsIdentifierBinding,
+    JsModuleItemList, JsSyntaxKind, SyntaxNodeText, TsMethodSignatureTypeMember,
+};
+use biome_rowan::{AstNode, TextRange};
+
+declare_rule! {
+    /// Disallow non adjacent overload signatures.
+    ///
+    /// Overload signatures must be adjacent.
```

Comment:
The rule is about enforcing adjacent signatures, however the rule is called `no*`. Should we call the rule `useAdjacentSignatures`?

Reply:
@ematipico 
Yes, that's a nice idea!!!